UPGRADE-4.0.md

```diff
@@ -155,3 +155,6 @@ Method `SimplePager::getResults` is always returning an array
 
 ## RouteCollectionInterface
 `RouteCollection` implements `RouteCollectionInterface`.
+
+## SearchHandler
+When no Pager is found, `SearchHandler::search()` now return `null` instead of `false`.
```

Comment:
```suggestion
When no Pager is found, `SearchHandler::search()` returns `null`. Previously, it was returning `false`.
```

Reply:
IIUC, `SearchHandler::search()` returns `null` when there are no searchable filters, but it doesn't depend on the availability of a pager in any case.

Reply:
Indeed, should be something like this then
```
When there is no searchable filters, `SearchHandler::search()` returns `null`. Previously, it was returning `false`.
```

Can you update the note @tambait ?

Reply:
Updated